portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -54,11 +54,13 @@ public void doGet(HttpServletRequest request, HttpServletResponse response) thro
       Translate translate = TranslateOptions.getDefaultInstance().getService();
       Translation translation = translate.translate(commentText, Translate.TranslateOption.targetLanguage(languageCode));
       String translatedText = translation.getTranslatedText();
-      Comment comment = new Comment(translatedText, timestamp, null, id);
+      translatedText = translatedText.replace("&#39;", "'");
```

Comment:
Where did you find this conversion? I would guess that there are a lot more special characters than just `'` that might show up and be encoding, its gonna be nearly impossible to find them all yourself.  There is likely a library out there that can handle this conversion for you, though I don't know what off hand.
Sounds like this answer might be helpful: https://stackoverflow.com/questions/229015/encoding-conversion-in-java

But I also know how hard it is debugging this since you have to deploy each time, so this is definately good enough for an mvp

Reply:
Before I had this line, if the translated comments had an apostrophe inserted it would show up as the literal string &#39 (even after fixing the charset), so I did the quick replacement to get a normal looking comment section. I'll look into the stackoverflow post, because I'm sure there are other special characters not featured in my test comments that aren't being caught!